docs/Factory.md

```diff
@@ -129,3 +129,15 @@ fastify.listen(3000)
 ```
 
 Internally Fastify uses the API of Node core http server, so if you are using a custom server you must be sure to have the same API exposed. If not, you can enhance the sever instance inside the `serverFactory` function before the `return` statement.
+
+<a name="factory-case-sensitive"></a>
+### `caseSensitive`
+
+If set to `false`, all paths would be lowercased by `route()` API, and
+all URLs will be lowercased before matching, essentially disabling case
+sensitivity for paths.
```

Comment:
I misinterpreted this explanation until I started writing a comment. I think it would be better to write:

> By default, value equal to `true`, routes are registered as case sensitive. That is, `/foo` is **not** equivalent to `/Foo`. When set to `false`, routes are registered in a fashion such that `/foo` is equivalent to `/Foo` which is equivalent to `/FOO`.